### PR TITLE
Brand Consolidation - Move the homepage banner below the navigation

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This pull request moves the homepage banner from above the preview banner to just below the navigation. There was some confusion originally, because the banner was referred to as the "top of page" banner, but that was a reference to the CSS selector and not accurate as to the location on the page.

Because VA.gov does not use the `#top-of-page-alert` style, I think we can delete all of that style post-launch, in the Design System's shame file, https://github.com/department-of-veterans-affairs/design-system/blob/e3001be1924ccbf935708cd6391c7910cf2ffefa/src/sass/_shame.scss#L256.

Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14758

## Testing done
1. Flipped the banner's visibility to true, https://github.com/department-of-veterans-affairs/vagov-content/blob/master/fragments/home/banner.yml#L1
2. Confirmed that it looks okay

## Screenshots
I followed the Vets.gov hurricane banner style, which omitted the left border.

screen shot 2018-11-02 at 12 27 58 pm

## Acceptance criteria
- [ ] Banner is in the right place and looks okay

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/sass/merger.scss [hunk 1]
```diff
@@ -825,8 +825,12 @@ img[data-srcset] {
...
-.usa-alert {
-  .top-of-page {
-    margin-top: 0;
+
+.usa-alert-full-width {
+  background-color: $color-gray-lightest;
+  .usa-alert {
+    border-left: none;
+    max-width: 1000px;
+    margin: 0 auto;
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/merger.scss]: Would this be better off in Formation?

Comment (jbalboni) [reply src/platform/site-wide/sass/merger.scss]: This feels kind of homepage/header specific to me, not something we could really reuse easily.

File: src/platform/site-wide/sass/merger.scss [hunk 2]
```diff
@@ -825,8 +825,12 @@ img[data-srcset] {
   padding-right: 1.2em;
   position: relative;
 }
-.usa-alert {
-  .top-of-page {
-    margin-top: 0;
+
+.usa-alert-full-width {
+  background-color: $color-gray-lightest;
+  .usa-alert {
```
Comment (jbalboni) [inline src/platform/site-wide/sass/merger.scss]: Could do `> .usa-alert` to make this less broad.

**Review by jbalboni — APPROVED** [review]

**Comment by rtluu** [comment]
@emilywaggoner and @smoonpeyes I'm with @andrewlewandowski right now and he'd like some design input on the removal of the colored border. Since we changed the background colors of our alerts to gray and the icons are now black, there is no color indicator

File: va-gov/includes/header.html [hunk 3]
```diff
@@ -121,5 +112,15 @@ <h3 class="usa-alert-heading">You're viewing a preview of the new VA.gov</h3>
   {% include "va-gov/includes/top-nav.html" %}
 {% endif %}
 
+{% if path == '' and banner and banner.visible %}
+  <div class="usa-alert-full-width">
+    <div aria-live="polite" role="alert" class="usa-alert usa-alert-{{ banner.type }}">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">{{ banner.title }}</h3>
+        {{ banner.content }}
```
Comment (U-DON) [inline va-gov/includes/header.html]: This should be wrapped in a `<div class="usa-alert-text">`.

**Review by U-DON — APPROVED** [review]
LGTM. Note that if we do end up wanting a different color background, like in the old alerts (light blue, yellow, etc.), you can now add a `background-color-only` class to the `usa-alert` div.

**Comment by smoonpeyes** [comment]
@rtluu @andrewlewandowski 

We tried adding a top border the banner, and I think it makes it stand out a little more now that the background is light grey. See examples below.

cc: @emilywaggoner 

**Informational**
global alert blue border - info

**Disaster**
global alert gold border - info

**Comment by andrewlewandowski** [comment]
Love this design solution. @smoonpeyes, @emilywaggoner, and @jbalboni - Does this mean that these two banners are, in fact, new patterns in our design system? So regular alerts still get the vertical colored bar on the left, but global header alerts get the top horizontal colored bar?

**Comment by andrewlewandowski** [comment]
@jbalboni - Okay, thanks for the information.

@emilywaggoner or @smoonpeyes  - Can you please add a ticket to the brand consolidation backlog to add this into Formation/design system? Thank you!

Cc: @chrisj-usds , @JeffBarnesUSDS

**Comment by jbalboni** [comment]
@andrewlewandowski @smoonpeyes I added the top border and updated the screenshot in the description of this PR. Let me know if that looks ok.

**Comment by smoonpeyes** [comment]
@jbalboni Looks great!

File: src/platform/site-wide/sass/merger.scss [hunk 4]
```diff
@@ -825,8 +825,18 @@ img[data-srcset] {
...
+    border-left: none;
+    max-width: 1000px;
+    margin: 0 auto;
+  }
+  &-warning {
+    border-top: 1rem solid $color-gold;
+  }
+  &-info {
+    border-top: 1rem solid $color-primary-alt-dark;
   }
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/merger.scss]: Should we also have the error type? Seems unlikely we'll ever need it, but 🤷‍♂️

**Comment by ncksllvn** [comment]
We'll migrate to the design system post-launch - thanks @smoonpeyes for writing up the ticket! Going to merge this once tests check out.

**Pull request merged**